### PR TITLE
[WIP] Add contact

### PR DESCRIPTION
**Pull request by ns5001**
Will be working on it

### CODE REVIEW

**Comment by ns5001** [comment]
@bibikhadiza @francisrocco @cliffregis @mahatikg 
Added a contact us link to main page, in addition to adding a contact us page

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,12 @@
 source 'https://rubygems.org'
 
-ruby '2.2.2'
+ruby '2.3.1'
```
Comment (kytrinyx) [inline Gemfile]: Please don't change the ruby version.

File: Gemfile [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
 gem 'activesupport', '~> 4.2.1'
 gem 'activerecord', '~> 4.2.1'
 
 gem 'bugsnag'
+gem 'shotgun'
```
Comment (kytrinyx) [inline Gemfile]: Please don't add development dependencies in an unrelated pull request.

File: app/routes/contact.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module ExercismWeb
+  module Routes
+  
+  end
+end
```
Comment (kytrinyx) [inline app/routes/contact.rb]: This file doesn't appear to be used.

File: app/views/contact/contact.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<h1>Contact Us</h1>
+<br>
+<br>
+
+<p>To report a bug, suggest improvements to exercism.io, or if you're having trouble
+installing or using the CLI, please <a href="https://github.com/exercism/exercism.io/issues">open a GitHub Issue.</a> You can also visit our <a href="https://gitter.im/exercism/support">support chat.</a>
+We'll do our best to help you out!</p>
+
+<p>Follow us on Twitter <a href="https://twitter.com/exercism_io">@exercism_io.</a></p>
```
Comment (kytrinyx) [inline app/views/contact/contact.erb]: I would put the period after the closing `</a>`.

File: app/views/contact/contact.erb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+<p>For occasional updates, such as new language tracks being launched,
+<a href="https://tinyletter.com/exercism">sign up for the newsletter.</a></p>
```
Comment (kytrinyx) [inline app/views/contact/contact.erb]: Same here—I'd put the period after the closing `</a>`.

**Comment by kytrinyx** [comment]
Thank you for working on this, @ns5001!

**Comment by kytrinyx** [comment]
I'm closing this, since it's been 3 weeks without a response from the submitter.

**Pull request closed without merging**